share/spice/similar_sites/content.handlebars

```diff
@@ -0,0 +1,22 @@
+<div id="container">
+    {{#list results 0 5}}{{/list}}
+
+    {{#if show_more}}
+    <div id="hidden">
+        {{#list results 6 0}}{{/list}}
+    </div>
+    {{/if}}
+
+    <div id="more_info">
+        {{#if show_more}}
+            <span class="tx-clr--lt ddgsi-plus" id="show_more">Show More</span>
+
+            <span class="vertical_roller"></span>
+        {{/if}}
+
+        <span id="more_at">
+            <img src="http://icons.duckduckgo.com/ip/www.similarsitesearch.com.ico" width="16px" height="16px"></img>
```

Comment:
@Adman could you change to `https`? We get mixed content otherwise. :)
![screen shot 2015-03-02 at 10 06 11 am](https://cloud.githubusercontent.com/assets/81969/6443271/ca065448-c0c3-11e4-931b-a36cbcecdb09.png)